src/screens/PinScreen.js

```diff
@@ -129,6 +131,12 @@ class PinScreen extends React.Component {
       // in case it's the lock screen, we just have to execute the data migration
       // method an change redux state. No need to execute callback or go back on navigation
       this.handleDataMigration(pin);
+      if (!this.props.wallet) {
+        // We are saving HathorWallet object in redux, so if the app has lost redux information and is in locked screen
+        // we must start the HathorWallet object again
+        const words = hathorLib.wallet.getWalletWords(pin);
```

Comment:
Shouldn't we create a `getWalletWords()` in the utils and call this method from there? I feel we should put all direct access to the walletLib in the same place for future refactor.

Reply:
Makes sense, done that.